src/Features/ScreenSpaceShadows.cpp

```diff
@@ -249,12 +258,16 @@ void ScreenSpaceShadows::ModifyLighting(const RE::BSShader*, const uint32_t)
 
 					data.RcpBufferDim.x = 1.0f / data.BufferDim.x;
 					data.RcpBufferDim.y = 1.0f / data.BufferDim.y;
+					if (REL::Module::IsVR())
+						data.ProjMatrix = shadowState->GetVRRuntimeData().cameraData.getEye().projMat;
+					else
+						data.ProjMatrix = shadowState->GetRuntimeData().cameraData.getEye().projMat;
 
-					data.ProjMatrix = shadowState->m_CameraData.m_ProjMat;
-					data.InvProjMatrix = XMMatrixInverse(nullptr, shadowState->m_CameraData.m_ProjMat);
+					data.ProjMatrix = shadowState->GetRuntimeData2().cameraData.getEye().projMat;
```

Comment:
As noted in the other PR, not sure where this is coming from, but it can be deleted as data.ProjMatrix is set above.